### ISSUE TITLE
Can not build batteries-included jar file

### ISSUE THREAD

**Issue by dblezek** [Mayo-AAU/HadoopCommons#11]

**Comment by dblezek** [comment]
Have an error that may need a Maven expert to solve.  Trying to build the batteries-included, 'uber-jar' to try testing on the cluster.  Got these errors:

```
mvn assembly:single
<snip>
[WARNING] Cannot include project artifact: edu.mayo:hadoop-commons:jar:1.0-SNAPSHOT; it doesn't have an associated file or directory.
```

Then when running the `.jar`:

```
11:53 AM]{mra9161:R5060116(local_testing_fix)}:HadoopCommons
21: java -jar target/hadoop-commons-1.0-SNAPSHOT-jar-with-dependencies.jar
Error: Could not find or load main class edu.mayo.hadoop.commons.examples.ListTables
```

Help?

**Comment by dblezek** [comment]
Google is my friend.  The correct command includes the `deploy` target:

```
mvn deploy assembly:single
```

However, it seems to be a ["fundamental](http://stackoverflow.com/a/574650/334619) and [unfixable issue"](http://stackoverflow.com/questions/14276453/maven-assembly-include-the-current-project-jar-in-the-final-zip-tar) that these commands need to be on the same command line, and do not work if run independently.  See [Jira for the details](https://issues.apache.org/jira/browse/MASSEMBLY-94).

**Comment by dblezek** [comment]
Fixed in baaba38
